wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsDFUClient.java

```diff
@@ -953,12 +953,30 @@ public void deleteSuperFileSubfiles(String superfile, List<String> subfiles) thr
      *            - the filename to search for
      * @param cluster
      *            - the cluster to search on
-     * @return - true if the file exists on the specified cluster (or on any cluster if the input cluster is null),
-     *         false otherwise
+     * @return - collection of files matching the logicalfilename passed in
      * @throws Exception
      * @throws ArrayOfEspExceptionWrapper
      */
     public List<DFULogicalFileWrapper> searchFiles(String logicalFilename, String cluster) throws Exception, ArrayOfEspExceptionWrapper
+    {
+       return searchFiles(logicalFilename,cluster,null,null);   
+    }
+    /**
+     * searchFiles
+     *
+     * @param logicalFilename
+     *            - the filename to search for
+     * @param cluster
+     *            - the cluster to search on
+     * @param pagesize
+     *            - the page to begin on
```

Comment:
looks like the javadoc params might be incorrect.  I think 'pageStartFrom' should probably say what 'pagesize' is labeled.